src/GNDStk/v1.9/containers/Axis/test/Axis.test.cpp

```diff
@@ -75,60 +86,46 @@ SCENARIO( "Axis" ) {
 
     WHEN( "the data is moved" ) {
 
-      Node core = chunk();
-      Axis chunk( core );
-      Axis chunk2( core );
+      Node node;
+      node << chunk();
+      Axis chunk( node );
+      Axis chunk2( node );
 
       // move constructor
       Axis move( std::move( chunk ) );
 
-      // copy assignment
+      // move assignment
       Axis assign;
       assign = std::move( chunk2 );
 
-      THEN( "an Axis can be constructed and members can be tested" ) {
+      THEN( "the component can be constructed and members can be tested" ) {
 
         verifyChunk( move );
         verifyChunk( assign );
       } // THEN
     } // WHEN
   } // GIVEN
 
-  GIVEN( "invalid data for an Axis" ) {
+  GIVEN( "invalid data" ) {
 
     WHEN( "when the wrong node is used" ) {
 
+      Node node;
+      node << invalidName();
+
       THEN( "an exception is thrown" ) {
 
-        CHECK_THROWS( Axis( invalidName() ) );
+        CHECK_THROWS( Axis( node ) );
       } // THEN
     } // WHEN
   } // GIVEN
 } // SCENARIO
 
-Node defaultChunk() {
-
-  Node chunk;
-  chunk << "<axis/>";
-
-  return chunk;
-}
-
-void verifyDefaultChunk( const Axis& component ) {
+std::string chunk() {
 
-  CHECK( std::nullopt == component.index() );
-  CHECK( std::nullopt == component.label() );
-  CHECK( std::nullopt == component.unit() );
-
-  CHECK( defaultChunk() == Node( component ) );
-}
-
-Node chunk() {
-
-  Node chunk;
-  chunk << "<axis index=\"1\" label=\"energy_in\" unit=\"eV\"/>";
-
-  return chunk;
+  return
+R"***(<axis index="1" label="energy_in" unit="eV" />
```

Comment:
I like this format for writing short, one-liner string literals. Easy to read.

Reply:
Even the larger ones are going to be clearer. And this way, we won't need escape characters all the time.